### PR TITLE
feat: added docs for LMS

### PR DESCRIPTION
**Pull request by scmmishra**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/education/navigating-the-portal.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+> Note: The activity of the student is only recorded after the student clicks on the Next button.
+
+<img class="screenshot" alt="Portal Contents" src="{{docs_base_url}}/assets/img/education/lms/content-navigation.png">
+
+### Navigating Quizzes
+
+In case of a quiz, the student has to first submit the quiz, following which the result is computed and displayed, after which the student can navigate to the next content.
+
+<img class="screenshot" alt="Quiz Pass" src="{{docs_base_url}}/assets/img/education/lms/quiz-pass.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/navigating-the-portal.md]: even here the radio buttons seems glitchy on the right side
not to mention the turfs on either side of the page

File: erpnext_com/www/docs/user/manual/en/education/navigating-the-portal.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The next time a student visits the quiz page, based on the previous attempts the portal may or may not allow any quiz attempts.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/navigating-the-portal.md]: based on previous attempts, (comma)

File: erpnext_com/www/docs/user/manual/en/education/setting-up-lms.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!-- add-breadcrumbs -->
+# Learning Management System
+
+The education modules is bundled with a Learning Management System (LMS) out of the box. This allows institutes to publish their programs on their website. Programs can contain rich text articles, videos and even quizzes. The progress of individual students can be tracked through the desk as well as the portal.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/setting-up-lms.md]: Education module
add comma after videos

File: erpnext_com/www/docs/user/manual/en/education/setting-up-lms.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+<img class="screenshot" alt="LMS Home" src="{{docs_base_url}}/assets/img/education/lms/home.png">
+
+LMS is disabled by deafult and can be enabled from Education Settings.
+
+To enable LMS go to :
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/setting-up-lms.md]: go to:

File: erpnext_com/www/docs/user/manual/en/education/setting-up-lms.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+> Education > Eduaction Settings > LMS Settings
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/setting-up-lms.md]: use bold not \>

File: erpnext_com/www/docs/user/manual/en/education/tracking-progress.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# Tracking the Progress
+
+The students can view their progress on the portal itself and do not have access to the desk. For instructors and other users of ERPNext, the following doctypes are used to track the progress of the student
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/tracking-progress.md]: colon at the end

File: erpnext_com/www/docs/user/manual/en/education/tracking-progress.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- Program Enrollment
+- Course Enrollment
+- Course Activity
+- Quiz Activity
+
+---
+
+## Program Enrollment
+
+If *Allow Self Enroll* is enabled for a particular program, an entry is created automatically on behalf of the student. Each student will have only one program enrollment for each program. You can learn more about program enrollment [here]({{ docs_base_url }}/user/manual/en/education/program-enrollment)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/tracking-progress.md]: ' ' not italics

File: erpnext_com/www/docs/user/manual/en/education/tracking-progress.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+<img class="screenshot" alt="Program Enrollment" src="{{docs_base_url}}/assets/img/education/lms/desk-program-enrollment.png">
+
+---
+
+## Course Enrollment
+
+For a particular course in a program, a course enrollment record is automatically created for each course as shown below.
+<img class="screenshot" alt="Course Enrollment List" src="{{docs_base_url}}/assets/img/education/lms/desk-course-enrollment-list.png">
+
+For a particular program and it's child course, only one course enrollment is created for a student. In case a course is added to the program later, the student will automatically be enrolled to the course when the student visits the portal next.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/tracking-progress.md]: its child course

File: erpnext_com/www/docs/user/manual/en/education/tracking-progress.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+For each non-quiz type content in a course, a course activity is created every time the student navigates through a content. This activity is created only once per content.
+
+<img class="screenshot" alt="Course Activity List" src="{{docs_base_url}}/assets/img/education/lms/desk-course-activity-list.png">
+<img class="screenshot" alt="Course Activity" src="{{docs_base_url}}/assets/img/education/lms/desk-course-activity.png">
+
+---
+
+## Quiz Activity
+
+For each quiz attempt, till the student is allowed to attempt the quiz, a quiz activity is created. This doctype has all the information about the attempt viz. The selected options for each question, attepmt date, as well as the result of the quiz.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/education/tracking-progress.md]: typo: attempt

File: erpnext_com/www/docs/user/manual/en/education/lms-masters.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
+# Setting up Masters
+
+LMS uses the following masters to show content on the portal.
+
+<img class="screenshot" alt="LMS Masters" src="{{docs_base_url}}/assets/img/education/lms/lms-masters-tree.png">
```
[the remark this answers is not in the document]
Comment (scmmishra) [reply erpnext_com/www/docs/user/manual/en/education/lms-masters.md]: Can you send me the editable files for the image in the link?

**Comment by scmmishra** [comment]
@prasadarr I have made all the changes requested

**Pull request merged**